test/benchmarks/driver_bench/src/suites/multi_bench/aggregate_a_million_documents_and_to_array.mts

```diff
@@ -0,0 +1,38 @@
+import { driver, type mongodb } from '../../driver.mjs';
+
+export const taskSize = 16;
+
+let db: mongodb.Db;
```

Comment:
It may look odd that our types are namespace access like this. Unlike JS we can't dynamically obtain types (of course) but we can point the "mongodb" type at a different .d.ts file if it become necessary. Currently, it is pointed at the one in the top level of the repo that's built from this driver. 

You can imagine a day where we want to support benchmarking different versions with different APIs, in order to keep our code type-safe, we can encode the differences into a .d.ts file such that benchmark code is still written safely.